docs/contributing/minutes.md

```diff
@@ -16,6 +16,88 @@ Attendees:
 
 ---
 
+## 28-09-2023
+
+Attendees: Karen , Daniel
+
+### ❗Actions from last time
```

Comment:
I think this and the "Actions" heading are maybe swapped? :)

Reply:
This is just hanging around